third_party/2and3/werkzeug/middleware/proxy_fix.pyi

```diff
@@ -0,0 +1,23 @@
+from typing import Iterable, Optional
+from wsgiref.types import StartResponse, WSGIApplication, WSGIEnvironment
+
+class ProxyFix(object):
+    app: WSGIApplication
+    x_for: int
+    x_proto: int
+    x_host: int
+    x_port: int
+    x_prefix: int
+    num_proxies: Optional[int]
```

Comment:
Actually just `int`; it's a special property that always returns the value of `x_for`.